packages/round-manager/src/features/api/__tests__/utils.test.ts

```diff
@@ -117,4 +118,35 @@ describe("graphql_fetch", () => {
       { id: "0x123456789544fe81379e2951623f008d200e1d18" }
     )
   })
+
+  it("should fetch data from the correct graphql endpoint for optimism-kovan network", async () => {
+    fetchMock.resetMocks()
+
+    fetchMock.mockResponseOnce(JSON.stringify({
+      data: {}
+    }))
+
+    await graphql_fetch(`programs { id }`, "optimism-kovan")
+
+    expect(fetchMock).toHaveBeenCalledWith(
+      `${process.env.REACT_APP_SUBGRAPH_OPTIMISM_KOVAN_API}`,
+      expect.anything()
+    )
+  })
+
+  it("should fetch data from the correct graphql endpoint for optimism network", async () => {
+    fetchMock.resetMocks()
+
+    fetchMock.mockResponseOnce(JSON.stringify({
+      data: {}
+    }))
+
+    await graphql_fetch(`programs { id }`, "optimism")
+
+    expect(fetchMock).toHaveBeenCalledWith(
+      //`${process.env.REACT_APP_SUBGRAPH_OPTIMISM_API}`,
```

Comment:
Do we need this line ?

Reply:
Thanks for catching - I can remove until we have a subgraph set up for optimism mainnet 👍